### ISSUE TITLE
Bash: add a few more builtins

### ISSUE THREAD

**Issue by adrian5** [nvim-treesitter/nvim-treesitter#2732]
I think these two should be added to [`@functions.builtin`](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/bash/highlights.scm#L103-L105):

```shell
source
test
```

Others to consider (maybe not if we target mostly regular scripts, and not bashrc):

```shell
alias
clear

# Too niche probably; I imagine there are several more like this
stty
shopt
```

**Comment by kyazdani42** [comment]
stty is a gnu linux based program so i don't think it should be included in bash builtins ;) otherwise i think you can add a PR for the others !

**Comment by adrian5** [comment]
So I'd just add them to that list and that's it?

**Comment by kyazdani42** [comment]
closed in #2733
